src/ape/types/signatures.py

```diff
@@ -1,11 +1,11 @@
 from typing import Iterator, Union
 
-from dataclassy import dataclass
 from eth_account.messages import SignableMessage
 from eth_utils import to_bytes
+from pydantic.dataclasses import dataclass
 
 
-@dataclass(frozen=True, slots=True, kwargs=True)
```

Comment:
I think the pydantic `dataclass` still supports `frozen`.
https://pydantic-docs.helpmanual.io/usage/dataclasses/

^ Disregard if we are intentionally getting rid of that